pkgs/development/php-packages/swoole/default.nix

```diff
@@ -10,9 +18,19 @@ in buildPecl {
     owner = "swoole";
     repo = "swoole-src";
     rev = "v${version}";
-    sha256 = "sha256-xadseYMbA+llzTf9JFIitJK2iR0dN8vAjv3n9/e7FGs=";
+    hash = "sha256-xadseYMbA+llzTf9JFIitJK2iR0dN8vAjv3n9/e7FGs=";
```

Comment:
The patch applies only because you did not bump the hash here so you are still building 5.0.3.